package/dropbear/0004-libdropbear.h-added.patch.conditional

```diff
@@ -0,0 +1,31 @@
+--- a/libdropbear.h	2019-08-28 20:32:09.684490106 +0530
```

Comment:
Patch needs appropriate format-patch header, including commit log providing a reason for the patch. For more info see: https://git-scm.com/docs/git-format-patch

Reply:
Yes, made the corrections and now created the patch with git format-patch header.